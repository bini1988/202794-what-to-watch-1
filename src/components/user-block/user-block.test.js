import React from "react";
import renderer from "react-test-renderer";
import UserBlock from "./user-block.jsx";

it(`UserBlock correctly renders default markup`, () => {
  const tree = renderer.create(
      <UserBlock
        user={{
          name: `User`,
          avatar: `/avatar.jpg`,
        }}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
