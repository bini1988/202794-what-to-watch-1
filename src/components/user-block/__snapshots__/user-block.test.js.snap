// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserBlock correctly renders default markup 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="User"
      height="63"
      src="/avatar.jpg"
      width="63"
    />
  </div>
</div>
`;
